single-line-diagram-core/src/main/java/com/powsybl/sld/layout/BlockOrganizer.java

```diff
@@ -34,6 +34,8 @@
 
     private final boolean exceptionIfPatternNotHandled;
 
+    private final boolean voltageIndicatorOnBus;
```

Comment:
voltageIndicator is only one possible use for busInfo, so we should keep that keyword for the test only

Reply:
maybe `addCellForBusInfo`, what do you think?